components/ProjectList/ProjectList.jsx

```diff
@@ -1,19 +1,95 @@
-import classNames from 'classnames/bind';
-import Project from '../Project/Project';
-
-import styles from './ProjectList.module.css';
+import classNames from "classnames/bind";
+import { groupBy } from "lodash";
+import { useEffect, useState } from "react";
+import { Activity } from "../Activity/Activity";
+import { Contributors } from "../Contributors/Contributors";
+import Project from "../Project/Project";
+import styles from "./ProjectList.module.css";
 
 const cx = classNames.bind(styles);
 
-/** @param {{ projectsData: import("../../types/projects-config").Project[] }} */
 export default function ProjectList({ projectsData }) {
```

Comment:
The idea of decomposition.

It seems that `ProjectList` is a separate component with a list of repositories.
Maybe we'll make some `<Dashboard>` that would consist of heading and 3 areas with:
   - Activity
   - Pulse
   - ProjectList

Reply:
We already have similar structure. Renamed ProjectList to Dashboard and added the structure: Dashboard -> that consists of Activity, Contributors, Projects